umm-spec-lib/test/cmr/umm_spec/test/validation/related_url.clj

```diff
@@ -77,6 +86,8 @@
        :errors ["[http://ingrid.ldgo.columbia.edu/SOURCES/.IGOSS/.fsu/|u|u+|u|v/dods] is not a valid URL"]}
       {:path [:RelatedUrls 0 :URL]
        :errors ["[http:\\\\fresc.usgs.gov\\products\\dataset\\moorhen_telemetry.zip] is not a valid URL"]}
+      {:path [:RelatedUrls 0]
```

Comment:
NIT: You have used index `0` throughout the tests for RelatedUrls. It would be nice to see an `1` or `2`.